spanner/unit_tests/test_client.py

```diff
@@ -114,7 +151,8 @@ def test_instance_admin_api(self):
         client = self._make_one(project=self.PROJECT, credentials=creds)
 
         class _Client(object):
-            pass
+            def __init__(self, *args, **kwargs):
+                pass
```

Comment:
This new constructor should record the values passed in, so that we can make assertions about them.